Sources/TuistCache/Utilities/CacheBundleBuilder.swift

```diff
@@ -45,6 +45,7 @@ public final class CacheBundleBuilder: CacheArtifactBuilding {
         try await xcodeBuildController.build(
             projectTarget,
             scheme: scheme.name,
+            destination: nil,
```

Comment:
To get this caching functionality to work correctly is it important that I use the `osVersion` and `deviceName` in this context to find the `XcodeBuildDestination`? My guess is actually yes but I wasn't sure.